__tests__/app.test.js

```diff
@@ -15,3 +23,31 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("2. GET /api/topics", () => {
+  test("2a. 200: responds with array of topcs including slug & description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(Array.isArray(topics)).toBe(true);
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("2b. 404: responds with error if route not found", () => {
+    return request(app)
+      .get("/api/notATopic")
```

Comment:
This test is great, but it should be in **it's own describe block**. The test is checking any incorrect route, so it's not specific to `/api/topics` , it's for the whole server.